driver-core/src/main/java/com/datastax/driver/core/policies/DowngradingConsistencyRetryPolicy.java

```diff
@@ -156,7 +156,7 @@ public RetryDecision onWriteTimeout(Statement statement, ConsistencyLevel cl, Wr
             case SIMPLE:
             case BATCH:
                 // Since we provide atomicity there is no point in retrying
-                return RetryDecision.ignore();
+                return requiredAcks > 0 ? RetryDecision.ignore() : RetryDecision.rethrow();
```

Comment:
@tolbertam you are right. I think we should reopen this pull request and rebase on top of #372 (JAVA-709) to benefit from the new test class for retry policies.